AngularJS/js/app.js

```diff
@@ -0,0 +1,51 @@
+var app = angular.module('wbooks',[]);
+
+/*
+Reference: http://jsfiddle.net/BB3JK/47/
+*/
+
+$('select').each(function(){
```

Comment:
This jQuery element is not necessary. You can have the elements and use `ng-repeat` https://docs.angularjs.org/api/ng/directive/ngRepeat to create the options and values in the `select`